tutorials/imad-tutorial-pt1/index.ipynb

```diff
@@ -0,0 +1,2078 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #17.            timestamp = im1.get('system:time_start').getInfo()
</pre><p> </p><p>You could shorted these time formatting steps using this:</p><p> </p><p><code>im1.date().format('E MMM dd HH:mm:ss YYYY')</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google/earthengine-community/pull/583/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>